config/routes.rb

```diff
@@ -8,6 +8,9 @@
   namespace 'api' do
     namespace 'v1' do
       resources :questions, only: :create
+      resources :conversations, only: :index do
+          resources :messages, only: :index, module: :conversations
```

Comment:
No need for `module: :conversations` here I believe

Reply:
`module: :conversations` is because the message controller is in api/v1/conversations/messages. 
Otherwise it will expect the controller to be in api/v1/messages

Reply:
Shouldn't the route be `api/v1/conversations/:conversation_id/messages` ?